modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs

```diff
@@ -42,74 +45,84 @@ public static partial class Mathf
             return (real_t)Math.Acos(s);
         }
 
         /// <summary>
-        /// Returns the arc sine of `s` in radians. Use to get the angle of sine s.
+        /// Returns the arc sine of <paramref name="s"/> in radians.
+        /// Use to get the angle of sine <paramref name="s"/>.
         /// </summary>
         /// <param name="s">The input sine value. Must be on the range of -1.0 to 1.0.</param>
-        /// <returns>An angle that would result in the given sine value. On the range `-Tau/4` to `Tau/4`.</returns>
+        /// <returns>
+        /// An angle that would result in the given sine value. On the range <c>-Tau/4</c> to <c>Tau/4</c>.
+        /// </returns>
         public static real_t Asin(real_t s)
         {
             return (real_t)Math.Asin(s);
         }
 
         /// <summary>
-        /// Returns the arc tangent of `s` in radians. Use to get the angle of tangent s.
+        /// Returns the arc tangent of <paramref name="s"/> in radians.
+        /// Use to get the angle of tangent <paramref name="s"/>.
         ///
         /// The method cannot know in which quadrant the angle should fall.
-        /// See <see cref="Atan2(real_t, real_t)"/> if you have both `y` and `x`.
+        /// See <see cref="Atan2(real_t, real_t)"/> if you have both <c>y</c> and <c>x</c>.
         /// </summary>
         /// <param name="s">The input tangent value.</param>
-        /// <returns>An angle that would result in the given tangent value. On the range `-Tau/4` to `Tau/4`.</returns>
+        /// <returns>
+        /// An angle that would result in the given tangent value. On the range <c>-Tau/4</c> to <c>Tau/4</c>.
+        /// </returns>
         public static real_t Atan(real_t s)
         {
             return (real_t)Math.Atan(s);
         }
 
         /// <summary>
-        /// Returns the arc tangent of `y` and `x` in radians. Use to get the angle
-        /// of the tangent of `y/x`. To compute the value, the method takes into
+        /// Returns the arc tangent of <paramref name="y"/> and <paramref name="x"/> in radians.
+        /// Use to get the angle of the tangent of <c>y/x</c>. To compute the value, the method takes into
         /// account the sign of both arguments in order to determine the quadrant.
         ///
         /// Important note: The Y coordinate comes first, by convention.
         /// </summary>
         /// <param name="y">The Y coordinate of the point to find the angle to.</param>
         /// <param name="x">The X coordinate of the point to find the angle to.</param>
-        /// <returns>An angle that would result in the given tangent value. On the range `-Tau/2` to `Tau/2`.</returns>
+        /// <returns>
+        /// An angle that would result in the given tangent value. On the range <c>-Tau/2</c> to <c>Tau/2</c>.
+        /// </returns>
         public static real_t Atan2(real_t y, real_t x)
         {
             return (real_t)Math.Atan2(y, x);
         }
 
-        /// <summary>
```

Comment:
This doesn't seem right...

Reply:
Yes, this was removed by mistake.

Reply:
Thanks, I missed this when cherry picking. I checked and found a few other mistakes, should be fixed now.